src/main/java/com/minecolonies/core/event/EventHandler.java

```diff
@@ -114,6 +118,18 @@ public class EventHandler
      */
     private static Map<UUID, ChunkPos> playerPositions = new HashMap<>();
 
+    /**
+     * Cache of loot table -> crops.
+     */
+    private static Map<ResourceLocation, List<MinecoloniesCropBlock>> cropDrops;
+
+    // stolen from BlockLootSubProvider
+    private static final LootItemCondition.Builder HAS_SILK_TOUCH = MatchTool.toolMatches(ItemPredicate.Builder.item().hasEnchantment(new EnchantmentPredicate(Enchantments.SILK_TOUCH, MinMaxBounds.Ints.atLeast(1))));
```

Comment:
I think we could have those in some utils or constants place. Seems in general useful even outside of this context. I feel we use some of these already somewhere